packages/connection-form/src/components/connect-form.tsx

```diff
@@ -27,6 +27,7 @@ import ConnectFormActions from './connect-form-actions';
 import { useConnectForm } from '../hooks/use-connect-form';
 import { validateConnectionOptionsErrors } from '../utils/validation';
 import SaveConnectionModal from './save-connection-modal';
+import { usePreference } from 'compass-preferences-model';
```

Comment:
Does this prohibit this package from being used in a non-Compass setting? Thinking about if we want to use it in VSCode or other web apps down the line.